demo/index.html

```diff
@@ -37,7 +37,7 @@
       </li>
     </template>
   </div>
-  <script src="kinto.dev.js"></script>
+  <script src="http://mozilla-services.github.io/kinto.js/kinto.dev.js"></script>
```

Comment:
> Shall we use https for that?

Good point; even better, using `//` for automatic current protocol conservation :)